assets/tools/rad-validate

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+echo "* locating oc command"
+if ocloc=$(which oc) &> /dev/null; then
```

Comment:
nit: some distros don't have the `which` command present, what about using `type` as a fallback?
something like

`if ocloc=$(which oc || type -p oc) &> /dev/null; then`

Reply:
thanks! i will change this to use `command`